tests/go.sum

```diff
@@ -131,10 +131,10 @@ github.com/pkg/errors v0.9.1/go.mod h1:bwawxfHBFNV+L2hUp1rHADufV3IMtnDRdf1r5NINE
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4/go.mod h1:xMI15A0UPsDsEKsMN9yxemIoYk6Tm2C1GtYGdfGttqA=
-github.com/rancher-sandbox/ele-testhelpers v0.0.0-20240911133917-d4312809d5eb h1:Q4XjVA19KEX/EFHzbsN+A0bsSac70MvAYDF2qJddF1I=
-github.com/rancher-sandbox/ele-testhelpers v0.0.0-20240911133917-d4312809d5eb/go.mod h1:Ex+a/ng4u2BvcGQdQjTHI48h88bQ6k2a7q8rnvU0XbQ=
 github.com/rancher-sandbox/ele-testhelpers v0.0.0-20240926133955-76d56ed44bbc h1:ZmyPqOhMftc9uDvE6lmwO5x67WHRseWhFbayUoVvBB4=
```

Comment:
I think you may delete this line, it is duplicate IMO

Reply:
It was added by `go mod tidy`, I think

Reply:
Okay